### DISCUSSION TITLE
Exercises to learn Miller

### DISCUSSION THREAD

**Discussion by GuilloteauQ** [johnkerl/miller#1647] [Show and tell]
Hi all! 
I crafted some small exercises for newcomers (like me) to learn Miller's features: https://github.com/GuilloteauQ/miller-exercises
I thought it might be useful to share.
Solutions might not be "perfect" as I am still new to Miller...

Thanks for the great work on Miller, it is *really* meeting one of my needs 🙂

**Comment by aborruso** [comment]
Great idea, thank you very much
______
Scusami per la brevità, ti sto scrivendo dal cellulare.

website: https://medium.com/tantotanto
38° 7' 48" N, 13° 21' 9" E EPSG:4326

Il gio 12 set 2024, 23:07 Quentin Guilloteau ***@***.***> ha
scritto:
